### PR TITLE
CI

### PR DESCRIPTION
**Pull request by jmfayard**

### CODE REVIEW

**Comment by jmfayard** [comment]
Build failed

**Comment by jmfayard** [comment]
Build failed

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,29 +1,21 @@
...
     steps:
       - uses: actions/checkout@v1
-      - name: Run a one-line script
-        run: echo Hello, world!
       - uses: actions/setup-java@v1
         with:
           java-version: 8
       - uses: eskatos/gradle-command-action@v1
         with:
           arguments: testDebugUnitTest
```
Comment (eskatos) [inline .github/workflows/main.yml:15]: You miss an `id` on the step that runs Gradle so the build scan output can be referenced in the commenting step:
```suggestion
          arguments: testDebugUnitTest
          id: gradle
```
See https://github.com/eskatos/gradle-command-action/issues/10#issuecomment-626306465

**Comment by jmfayard** [comment]
Thanks @eskatos

**Comment by jmfayard** [comment]
Build failed

**Pull request merged**